From: Debian Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 11 Oct 2018 14:24:16 +0300
Subject: Comment out error message

===================================================================
---
 kdm/kfrontend/genkdmconf.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kdm/kfrontend/genkdmconf.c b/kdm/kfrontend/genkdmconf.c
index 66e516e..f6ff3d2 100644
--- a/kdm/kfrontend/genkdmconf.c
+++ b/kdm/kfrontend/genkdmconf.c
@@ -3153,9 +3153,11 @@ int main(int argc, char **argv)
                 memcmp(oldkde + olen - sizeof(SHR_CONF) + 1,
                        SHR_CONF, sizeof(SHR_CONF)))
             {
+                /*
                 fprintf(stderr,
                         "Warning: --old-kde does not end with " SHR_CONF ". "
                         "Might wreak havoc.\n");
+                */
                 oldkdepfx = oldkde;
             } else
                 ASPrintf((char **)&oldkdepfx,
