From: Debian Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 11 Oct 2018 14:24:17 +0300
Subject: Honour user KDM themeing

---
 kdm/config.def | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kdm/config.def b/kdm/config.def
index e219d42..0cb8337 100644
--- a/kdm/config.def
+++ b/kdm/config.def
@@ -2688,7 +2688,7 @@ Key: Theme
 Type: string
 Default: ""
 User: greeter
-Instance: */KDMDATA "/themes/elarun"
+Instance: */ "@@@ToBeReplacedByDesktopBase@@@"
 Comment: &
 Description:
  The theme to use for the greeter. Can point to either a directory or an XML
