From: Debian Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 11 Oct 2018 14:24:17 +0300
Subject: kdm_xreset_hook_framework

---
 kdm/kfrontend/genkdmconf.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/kdm/kfrontend/genkdmconf.c b/kdm/kfrontend/genkdmconf.c
index a1ebb35..ca868a2 100644
--- a/kdm/kfrontend/genkdmconf.c
+++ b/kdm/kfrontend/genkdmconf.c
@@ -701,7 +701,12 @@ static const char def_reset[] =
 "#chown root /dev/console\n"
 "#chmod 622 /dev/console\n"
 "\n"
-"# XDM configurations typically have sessreg here. KDM has it built-in.\n";
+"# XDM configurations typically have sessreg here. KDM has it built-in.\n"
+"\n"
+"# Use common Xreset framework if it exist\n"
+"if [ -x /etc/X11/Xreset ] ; then\n"
+"   /etc/X11/Xreset\n"
+"fi";
 
 static const char def_session1[] =
 "#! /bin/sh\n"
