From: Debian Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 11 Oct 2018 14:24:18 +0300
Subject: kubuntu_kcm_display_add_kscreen

===================================================================
---
 kcontrol/hardware/display/display.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kcontrol/hardware/display/display.cpp b/kcontrol/hardware/display/display.cpp
index 8746d09..079e685 100644
--- a/kcontrol/hardware/display/display.cpp
+++ b/kcontrol/hardware/display/display.cpp
@@ -41,7 +41,7 @@ KCMDisplay::KCMDisplay( QWidget *parent, const QVariantList & )
 {
   m_tabs = new QTabWidget( this );
 
-  addTab( "randr", i18n( "Size && Orientation" ) );
+  addTab( "kcm_kscreen", i18n( "Size && Orientation" ) );
   addTab( "nvidiadisplay", i18n( "Graphics Adaptor" ) );
   addTab( "nvidia3d", i18n( "3D Options" ) );
   addTab( "kgamma", i18n( "Monitor Gamma" ) );
