From: Debian Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 11 Oct 2018 14:24:18 +0300
Subject: kubuntu_no_krandr

===================================================================
---
 startkde.cmake | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/startkde.cmake b/startkde.cmake
index 9b832a7..ae3007f 100644
--- a/startkde.cmake
+++ b/startkde.cmake
@@ -70,12 +70,6 @@ kcminputrc Mouse cursorTheme 'Oxygen_White'
 kcminputrc Mouse cursorSize ''
 ksplashrc KSplash Theme Default
 ksplashrc KSplash Engine KSplashX
-krandrrc Display ApplyOnStartup false
-krandrrc Display StartupCommands ''
-krandrrc [Screen0]
-krandrrc [Screen1]
-krandrrc [Screen2]
-krandrrc [Screen3]
 kcmfonts General forceFontDPI 0
 kdeglobals Locale Language '' # trigger requesting languages from KLocale
 EOF
@@ -105,7 +99,8 @@ if test -n "$kcminputrc_mouse_cursortheme" -o -n "$kcminputrc_mouse_cursorsize"
     fi
 fi
 
-. krandrstartup
+# disabled due to move to kscreen - jriddell 2013-03
+# . krandrstartup
 
 if test "$kcmfonts_general_forcefontdpi" -ne 0; then
     xrdb -quiet -merge -nocpp <<EOF
