From: Debian Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 11 Oct 2018 14:24:18 +0300
Subject: kubuntu_avoid_zic_and_deep_copy_timezone_data

===================================================================
---
 kcontrol/dateandtime/helper.cpp | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/kcontrol/dateandtime/helper.cpp b/kcontrol/dateandtime/helper.cpp
index 5f6f35a..01fd73f 100644
--- a/kcontrol/dateandtime/helper.cpp
+++ b/kcontrol/dateandtime/helper.cpp
@@ -177,11 +177,20 @@ int ClockHelper::tz( const QString& selectedzone )
 #else
         QString tz = "/usr/share/zoneinfo/" + selectedzone;
 
-        QString zic = KStandardDirs::findExe("zic", exePath);
+        // Never check for zic as currently it is defunct on Debian systems.
+        // LP: #1212662
+        QString zic;
         if (!zic.isEmpty()) {
             KProcess::execute(zic, QStringList() << "-l" << selectedzone);
         } else if (!QFile::remove("/etc/localtime")) {
           ret |= TimezoneError;
+        } else if (!QFile::symLinkTarget(tz).isEmpty()) {
+          // If the timezone file is a symlink, get its link target and copy
+          // that instead as otherwise the symlink may end up being broken.
+          // LP: #1206199
+          if (!QFile::copy(QFile::symLinkTarget(tz), "/etc/localtime")) {
+            ret |= TimezoneError;
+          }
         } else if (!QFile::copy(tz, "/etc/localtime")) {
           ret |= TimezoneError;
         }
