From: Debian Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 11 Oct 2018 14:24:18 +0300
Subject: check_if_SensorMgr

===================================================================
---
 libs/ksysguard/ksgrd/SensorAgent.cpp | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/libs/ksysguard/ksgrd/SensorAgent.cpp b/libs/ksysguard/ksgrd/SensorAgent.cpp
index 69b978a..583c208 100644
--- a/libs/ksysguard/ksgrd/SensorAgent.cpp
+++ b/libs/ksysguard/ksgrd/SensorAgent.cpp
@@ -109,9 +109,10 @@ void SensorAgent::processAnswer( const char *buf, int buflen )
           else {
             /* We just received the end of an error message, so we
              * can display it. */
-            SensorMgr->notify( i18nc( "%1 is a host name", "Message from %1:\n%2",
-                               mHostName ,
-                               error ) );
+            if (SensorMgr)
+              SensorMgr->notify( i18nc( "%1 is a host name", "Message from %1:\n%2",
+                                        mHostName ,
+                                        error ) );
           }
           found = true;
 	  break;
